qtconsole/bracket_matcher.py

```diff
@@ -29,6 +29,9 @@ def __init__(self, text_edit):
         self.format = QtGui.QTextCharFormat()
         self.format.setBackground(QtGui.QColor('silver'))
 
+        self.format_unmatched = QtGui.QTextCharFormat()
+        self.format_unmatched.setBackground(QtGui.QColor('red'))
```

Comment:
Can we make this color configurable in the CSS stylesheet? It may clash with someone's colorscheme.